apps/src/blockly/addons/cdoXml.ts

```diff
@@ -216,7 +213,7 @@ export function addMutationToBehaviorBlocks(blockElement) {
     USER_CREATED_XML_ATTRIBUTE,
     FALSEY_DEFAULT
   );
-  mutationElement.setAttribute('userCreated', userCreated);
+  mutationElement.setAttribute('userCreated', `${userCreated}`);
```

Comment:
Nice catch! Can you confirm these should be strings because the `mutationElement` is still an XML element? (We're moving it from the block XML element to the mutation XML element?)

Reply:
they should be strings because all `setAttribute` values should be strings. It's a bit of a TypeScript annoyance, the function says [string](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#value) but it will coerce any value to a string. But TypeScript enforces string.